subprojects/plugins/src/main/java/org/gradle/api/plugins/jvm/JvmTestSuite.java

```diff
@@ -71,7 +71,7 @@
     /**
      * Get the test type for this test suite.
      *
-     * Defaults to {@link TestSuiteType#UNIT_TEST} for the built-in test suite, and to the kebab cased name of the
+     * Defaults to {@link TestSuiteType#UNIT_TEST} for the built-in test suite, and to the dash-case name of the
```

Comment:
Good catch, looks like I didn't use this anywhere else, though it does exist elsewhere in our repo.